eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/policy/dao/PolicyDefinitionEntityDAOImpl.java

```diff
@@ -17,14 +17,17 @@
 package org.apache.eagle.policy.dao;
 
 import org.apache.eagle.alert.entity.AbstractPolicyDefinitionEntity;
+import org.apache.eagle.alert.entity.AlertDefinitionAPIEntity;
```

Comment:
And the generic template parameter T is introduced for the purpose. I think we don't need to direct reference to AlertDefinitionAPIEntity here.